src/backend/.gitignore

```diff
@@ -1,2 +1,3 @@
 **/env/
```

Comment:
**/env/
**/__pycache__/
**/.cache
src/**.pyc
tests/**.pyc
.pytest_cache
__init__.pyc
**/**.pyc

Add this and then we don't have to have any other .gitignores